lib/deliver/commands_generator.rb

```diff
@@ -49,7 +49,17 @@ def run
           Deliver::Runner.new(options).run
         end
       end
-
+      command :submit_build do |c|
+        c.syntax = 'deliver submit_build'
+        c.description = 'Submit a specific build-nr for review, use latest for the latest build'
+        c.action do |args, options|
+          options = FastlaneCore::Configuration.create(Deliver::Options.available_options, options.__hash__)
+          options.load_configuration_file("Deliverfile")
+          options[:submit_for_review] = true
+          options[:build_number] = "latest" unless options[:build_number]
```

Comment:
well this won't work out - when build_number defaults to a value that is not `false` - it will trigger the submit_build code path - therefore i think it is neccassary to have this portion of ugliness here in the commander part - for keeping the rest of the implementation clean.